### PR TITLE
fix link to "inlining" documentation

### LINKED ISSUE #2030: Documentation needs better discussion of inlining (and link there is broken)
**Describe the bug**
I have noticed several broken links on the website docs so I am opening this issue about them. Please let me know if there is a better place for this.

First one is the FAQ references to inlining is broken

https://linkml.io/linkml/faq/modeling.html#how-do-i-represent-relationships-in-linkml

https://w3id.org/linkml/inlining

In general a think the docs could use a clearer discussion of inlining as I think I just have to try it out on a bunch of examples to understand what it means.

**To reproduce**

**Expected behavior**

**Screenshots**

**About your computer (if applicable, please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by nlharris**
fixes #2030

### CODE REVIEW

**Comment by nlharris** [comment]
I'm not sure why this failed the tests. I didn't change any code.

**Comment by sujaypatil96** [comment]
I re-ran the jobs that didn't go through, the logs were very strange as to why it wasn't going through just for Python 3.9/3.10 on ubuntu.

File: docs/faq/modeling.md [hunk 1]
```diff
@@ -618,6 +618,6 @@ Depending on the answer, LinkML has different modeling constructs to help you, i
 
 - [range](https://w3id.org/linkml/range) constraints, which can refer to classes
 - the ability to assign a slot as an [identifier](https://w3id.org/linkml/identifier), allowing other objects to link to it
-- [inlining](https://w3id.org/linkml/inlining) which determines how relationships are serialized in formats like JSON
+- [inlining](https://linkml.io/linkml/schemas/inlining.html), which determines how relationships are serialized in formats like JSON
```
Comment (sujaypatil96) [inline docs/faq/modeling.md:621]: Is this the best way to link out the document file, or is this:

```md
[inlining](../schemas/inlining.md)
```

Comment (nlharris) [reply docs/faq/modeling.md:621]: well, earlier in the same document it spells out the URL as i did. i'm fine to change it to a relative path, which might be more sustainable, but then we should do that consistently with the other links in the document.

Comment (nlharris) [reply docs/faq/modeling.md:621]: oh, actually, i think that should be
> [inlining](../schemas/inlining)

(no .md)

Comment (nlharris) [reply docs/faq/modeling.md:621]: @sujaypatil96 lmk what you want me to do, or feel free to edit this PR! i am not sure why it's failing tests.

Comment (sujaypatil96) [reply docs/faq/modeling.md]: Tests are not going through because of a recent PR/merge introducing `pyshacl` into the framework.

**Pull request merged**